frontend/app/components/Header/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link'
+import Logo from '../Logo'
+import CartButton from './CartButton'
+import Nav from './Nav'
+import {headerQuery} from '@/sanity/lib/queries'
+import {sanityFetch} from '@/sanity/lib/live'
+import {HeaderLinkGroups} from '@/types/types'
+import {urlForImage} from '@/sanity/lib/utils'
+import HeaderScrollController from './HeaderScrollController'
+
+export default async function Header() {
+  const {data: header} = await sanityFetch({
+    query: headerQuery,
+  })
+  return (
+    <HeaderScrollController>
+    <header className="fixed z-50 inset-x-0 top-0 bg-white m-4 sm:m-8 h-16 px-4 md:px-10 grid grid-cols-3 items-center gap-4 font-text transition-transform duration-400">
+      <Nav linkGroups={header?.linkGroups as HeaderLinkGroups | null} />
```

Comment:
Be careful using `as` since we are forcing a type upon something. When you encounter yourself using "as". Take a pause and ask yourself if there is another solution forward :)